### PR TITLE
Transaction tests

### LINKED ISSUE #80: Testing for transaction folder
As a DEVELOPER I want to check if views.py upload and get data from firestore correctly.

A/C Criteria:

- [ ] Tests have been written to test the validity of the views.py calls to firebase.

### PR DESCRIPTION
**Pull request by Chong-source**
Added tests testing the validity of firebase calls in views.py. Used mocked objects so that we don't have to call firebase API when we're running the tests.
Resolves #80

### CODE REVIEW

**Comment by jnnchi** [comment]
The tests look great, thank you!

File: app.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{
```
Comment (YoyoLiuuu) [inline app.json:1]: Thanks Jennifer for moving this to the right location!

File: backend/transaction/tests.py [hunk 2]
```diff
@@ -4,11 +4,127 @@
...
+        request = RequestFactory().get('/transaction/get')
+
+        # Call the function
+        response = get_data_from_firestore(request)
+
+        # Assertions
+        self.assertIsInstance(response, JsonResponse)
+        self.assertEqual(response.status_code, 500)
+        error_data = json.loads(response.content)
+        self.assertIn('error', error_data)
```
Comment (YoyoLiuuu) [inline backend/transaction/tests.py:127]: Looks great! Thanks for the great work Chong!

Comment (YoyoLiuuu) [reply backend/transaction/tests.py]: I kept the version in Transaction_test branch.

**Pull request merged**